diagrams/sequencediagram.txt

```diff
@@ -95,68 +93,89 @@ t_swap -> t_wallet : if Bob, SEND PENDING Msg Reveal (maker is sender)
 ==Changing semantics: On Commit-Reveal, Maker and Taker were the key roles. From now on Bob or Alice are the key roles. Now t_ is bob_ on the left and m_ is alice_ on the right.==
 ==Swap setup: Bob is left, Alice right==
 t_wallet -> t_database: Ctl CheckpointWalletBobPrelockBob
-t_wallet -> t_swap : Ctl CoreArbitratingSetup
+t_wallet -> t_swap : Msg CoreArbitratingSetup
+activate t_swap
 t_swap -> t_database: Ctl CheckpointSwapBobPrelockBob
 // TODO: During replay of CheckpointSwapBobPrelockBob, Bob has to rewatch these 3 txs
-t_syncer <- t_swap : Watch Arbitrating Lock
-t_syncer <- t_swap : Watch Cancel
-t_syncer <- t_swap : Watch Refund
+t_syncer <- t_swap : Sync Task(Watch Arbitrating Lock)
+t_syncer <- t_swap : Sync Task(Watch Cancel)
+t_syncer <- t_swap : Sync Task(Watch Refund)
 peerd <- t_swap : Msg CoreArbitratingSetup
+deactivate t_swap
 m_swap <- peerd : Msg CoreArbitratingSetup
-m_swap -> m_syncer : Watch Arbitrating Lock
+activate m_swap
+m_swap -> m_syncer : Sync Task(Watch Arbitrating Lock)
 // TODO: During replay of CheckpointWalletAlicePrelockBob, Alice has to rewatch these 2 txs (arbitrating already final then)
-m_swap -> m_syncer : Watch Cancel
-m_swap -> m_syncer : Watch Refund
+m_swap -> m_syncer : Sync Task(Watch Cancel)
+m_swap -> m_syncer : Sync Task(Watch Refund)
 
 m_wallet <- m_swap : Msg CoreArbitratingSetup
+deactivate m_swap
 m_wallet -> m_database : Ctl CheckpointWalletAlicePrelockBob
-m_wallet -> m_swap : Ctl RefundProcedureSignatures
+m_wallet -> m_swap : Msg RefundProcedureSignatures
+activate m_swap
 m_swap -> m_database : Ctl CheckpointSwapAlicePrelockBob
 m_swap -> peerd : Msg RefundProcedureSignatures
+deactivate m_swap
 peerd -> t_swap : Msg RefundProcedureSignatures
+activate t_swap
 t_wallet <- t_swap : Msg RefundProcedureSignatures
-t_wallet -> t_swap:Ctl Datum::SignedArbitratingLock
+deactivate t_swap
 // DONE: do we know that same inputs are being used in case of replay?
 // -> yes, but create different sig
 t_wallet -> t_database : Ctl CheckpointWalletBobPreBuySig
 t_wallet -> t_swap : Ctl Tx::Lock
-t_swap -> t_syncer: Ctl Broadcast Arbitrating Lock
-t_swap -> t_syncer : Watch Accordant Lock
+activate t_swap
+t_swap -> t_syncer: Sync Task(Broadcast Arbitrating Lock)
+t_swap -> t_syncer : Sync Task(Watch Accordant Lock)
+deactivate t_swap
+group sync messages
 t_wallet -> t_swap: Ctl Tx::Cancel
-t_wallet -> t_swap: Ctl Tx::Refund
-t_wallet -> t_swap : Ctl BuyProcedureSignature
+t_wallet ->(6) t_swap: Ctl Tx::Refund
```

Comment:
can you add a comment here that explains that the crossover of lines here signifies that ordering of the `Msg BuyProcedureSignature` wrt the other two messages on the `Ctl` bus is not guaranteed?